.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: cpp
```

Comment:
That really sucks :cry: 
Maybe we could still run it just for verification? 45 minutes is better than nothing (so we can catch osx mistakes). Your call though...